WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -198,15 +198,18 @@ decisions = {
 			age = 16
 			scaled_wealth = 0.5
 			prisoner = no
-			religion_group = christian
-			NOT = { has_character_flag = VIET_celtic_high_cross }
 			OR = {
-				culture = saxon
+				religion = insular
+				religion = pelagianism
 				AND = {
-					culture_group = celtic
-					NOT = { culture = breton }
+					religion_group = christian
+					OR = {
+						culture = saxon
+						culture = old_saxon
+					}
```

Comment:
Not Saxons, that is you misunderstanding.

Celtic rulers did by influence of Christian Celts raise such crosses.